ccm_web/client/src/components/FeatureCard.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react'
+import { makeStyles } from '@material-ui/core/styles'
+import Card from '@material-ui/core/Card'
+// import CardActions from '@material-ui/core/CardActions'
+import CardContent from '@material-ui/core/CardContent'
+import Grid from '@material-ui/core/Grid'
+import Typography from '@material-ui/core/Typography'
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    backgroundColor: '#FAFAFA'
+  },
+  centered: {
+    width: '100%'
+  },
+  title: {
+    fontSize: 14
+  },
```

Comment:
Yeah, some of this should be in a `theme` or the root CSS file. But you already know that!

Reply:
I feel the Home outline is missing a Layout. As things needs to be centered. It should have a consistence margin on top/bottom/left/right.